README.md

```diff
@@ -23,7 +23,7 @@ julia> x = rand(10)
 
 julia> @code_js identity(x)
 let model = (function () {
-  let math = dl.ENV.math;
+  let math = tf;
```

Comment:
Since this isn't saving us any characters any more, let's just use `tf` directly everywhere :)